src/poll.rs

```diff
@@ -17,11 +18,13 @@ use std::thread;
 use std::time::{Duration, Instant};
 use walkdir::WalkDir;
 
+#[derive(Debug)]
 struct PathData {
     mtime: i64,
     last_check: Instant,
 }
 
+#[derive(Debug)]
```

Comment:
This one isn't exposed, so Debug isn't necessary.

Reply:
This `Debug` impl (and `PathData`) is used by the `Debug` impl for `PollWatcher` (for the `.watches` field).

Would you prefer to leave the `.watches` field out of the Debug impl?

Reply:
it's fine, it does expose some inner workings, but debug impl isn't stable per semver AFAIK, so reducing the printed info if relevant should be fine